router/throttler/throttler.go

```diff
@@ -0,0 +1,156 @@
+package throttler
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/coinpaprika/ratelimiter"
+	"github.com/rudderlabs/rudder-server/config"
+	"github.com/rudderlabs/rudder-server/utils/logger"
+)
+
+var (
+	settings map[string]Settings
```

Comment:
Naming seems a bit odd.

If Settings is a type, then an instance of Settings should be settings. Change to settingsMap or allSettings?